docs/StandardLibraryProgrammersManual.md

```diff
@@ -171,6 +171,65 @@ The standard library utilizes thread local storage (TLS) to cache expensive comp
 
 See [ThreadLocalStorage.swift](https://github.com/apple/swift/blob/master/stdlib/public/core/ThreadLocalStorage.swift) for more details.
 
+
+## Working with Resilience
+
+Maintaining ABI compatibility with previously released versions of the standard library makes things more complicated. This section details some of the extra rules to remember and patterns to use.
+
+### The Curiously Recurring Inlinable Switch Pattern (CRISP)
```

Comment:
```suggestion
### The Curiously Recursive Inlinable Switch Pattern (CRISP)
```

Reply:
It's a reference to the [Curiously Recurring Template Pattern](https://en.wikipedia.org/wiki/Curiously_recurring_template_pattern), but maybe it doesn't need to be.

Reply:
Either name is fine by me, but you've used the "Curiously *Recursive* Inlinable Switch Pattern" elsewhere (in the final paragraph of the section; and in the commit/PR title).

Reply:
Ooh, whoops, thanks! Okay then.